packages/manager/src/constants.ts

```diff
@@ -151,6 +152,7 @@ export const dcDisplayNames = {
   'ca-east': 'Toronto, ON', // @todo Fallback for old Toronto ID; remove once DB has been updated.
   'ap-west': 'Mumbai, IN',
   'ap-southeast': 'Sydney, AU',
+  'us-iad': 'Washington DC',
```

Comment:
Should there be a comma?

```suggestion
  'us-iad': 'Washington, DC',
```